src/components/landing/hero.module.css

```diff
@@ -0,0 +1,22 @@
+.hero_section{
```

Comment:
@ibimina We have decided to use `camelCase`

Reply:
ok